### PR TITLE
ci: Setup scripts for Fedora

### LINKED ISSUE #79: Add a fedora CI setup script
Add a `.ci/setup_env_fedora.sh` script.

See: https://github.com/kata-containers/tests/pull/71#discussion_r165942902

### PR DESCRIPTION
**Pull request by GabyCT**
This will install Kata Containers on Fedora

Fixes #79

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

**Comment by chavafg** [comment]
lgtm

**Comment by jodh-intel** [comment]
lgtm

The new Jenkins CI has failed though. Is that expected at this stage @egernst?

File: .ci/install_runtime.sh [hunk 1]
```diff
@@ -33,7 +34,9 @@ NEW_RUNTIME_CONFIG="${PKGDEFAULTSDIR}/configuration.toml"
 clone_build_and_install "github.com/kata-containers/runtime"
 
 # Check system supports running Kata Containers
-kata-runtime kata-check
+if [ "$ID" == "ubuntu" ]; then
+	kata-runtime kata-check
```
Comment (jcvenegas) [inline .ci/install_runtime.sh]: why only check kata support for ubuntu ? could you add a comment if this is only need for this.

File: .ci/setup_env_fedora.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
```
Comment (jcvenegas) [inline .ci/setup_env_fedora.sh]: This is a new file right? I think here is just need 2018 @grahamwhaley  ?

**Comment by chavafg** [comment]
@jodh-intel Yes, it is expected. I will work today with @GabyCT to open issues (if not already opened) and start skipping tests to have a stable CI.

File: .ci/setup_env_fedora.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+set -e
+
+cidir=$(dirname "$0")
+source /etc/os-release
+source "${cidir}/lib.sh"
+get_cc_versions
+
+arch=$(arch)
+
+if [ "$arch" = x86_64 ]; then
```
Comment (jcvenegas) [inline .ci/setup_env_fedora.sh]: This is the same check for ubuntu , I wonder if you could move it to `setup.sh`  
https://github.com/kata-containers/tests/blob/master/.ci/setup_env_ubuntu.sh#L17

File: .ci/setup_env_fedora.sh (hunk 2 above)
Comment (grahamwhaley) [reply .ci/setup_env_fedora.sh]: A new file should just be 2018. If you copied the file from another (maybe another distro file), and then modified it, then 2017-2018 is fine.

**Comment by GabyCT** [comment]
@jcvenegas changes applied

**Comment by jcvenegas** [comment]
lgtm

**Pull request merged**